data_management/tests/test_api.py

```diff
@@ -369,19 +369,19 @@ def test_get_detail(self):
 
         self.assertEqual(response.status_code, 200)
         self.assertEqual(response['Content-Type'], 'application/json')
-        self.assertEqual(response.json()['doi_or_unique_name'], 'scottish deaths-involving-coronavirus-covid-19')
+        self.assertEqual(response.json()['other_unique_name'], 'scottish deaths-involving-coronavirus-covid-19')
 
     def test_filter_by_name(self):
         client = APIClient()
         client.force_authenticate(user=self.user)
         url = reverse('externalobject-list')
-        response = client.get(url, data={'doi_or_unique_name': '10.15586/jptcp.v27iSP1.691'}, format='json')
+        response = client.get(url, data={'other_unique_name': '10.15586/jptcp.v27iSP1.691'}, format='json')
```

Comment:
I'm not sure about what's going on here, but this would presumably be:
```suggestion
        response = client.get(url, data={'identifier': 'https://dx.doi.org/10.15586%2Fjptcp.v27iSP1.691'}, format='json')
```